README.md

```diff
@@ -205,24 +205,24 @@ Since version [0.6.0](https://github.com/cargo-generate/cargo-generate/releases/
 Here [an example](https://github.com/sassman/hermit-template-rs):
 
 ```toml
-[placeholders.hypervisor]
+[placeholders.HYPERVISOR]
```

Comment:
I feel not very contempt about this upper casing, but I had an idea how we can avoid it.

What if we would provide like for the templating context always the following 3 identical versions of a variable:

- named as originally provided (e.g. via `--define`)
- lower cased 
- upper cased

by doing so we would prevent the situation to have upper cased variables in the template but still we could provide by the environment variables that are upper cased.

Reply:
but is the problem not that a template author would like to work with template variables that then would work for all users on all platforms equally?

In order to guarantee this, we could also set out for the convention that template variables should always follow a lower case naming conventions, and we make sure that variable names provided via env variables are lower cased to fulfil the convention. In all other cases they are as users provide them. No further magic.

Reply:
> In order to guarantee this, we could also set out for the convention that template variables should always follow a lower case naming conventions, and we make sure that variable names provided via env variables are lower cased to fulfil the convention. In all other cases they are as users provide them. No further magic.

This I like :-)

A simple convention, a small fix to the implementation and mentioning it in the README... consider this one done!